libr/include/r_debug.h

```diff
@@ -186,30 +186,30 @@ typedef struct r_debug_t {
 	char *arch;
 	int bits; /// XXX: MUST SET ///
 
-	int pid;    	/* selected process id */
-	int tid;    	/* selected thread id */
-	int forked_pid;	/* last pid created by fork */
+	int pid; /* selected process id */
+	int tid; /* selected thread id */
+	int forked_pid; /* last pid created by fork */
 	RList *threads; /* XXX This is platform-specific !!! */
```

Comment:
the comment refers to the fact that the data stored in there is handled different on each platform, so it makes it not handy for external use.